enterprise_access/apps/api/v1/tests/test_subsidy_access_policy_views.py

```diff
@@ -1066,11 +1067,16 @@ def setup_mocks(self):
         self.addCleanup(contains_key_patcher.stop)
         self.addCleanup(get_content_metadata_patcher.stop)
 
+    @mock.patch('enterprise_access.apps.api_client.base_oauth.OAuthAPIClient')
```

Comment:
I get why/how this mock would be used but I'm curious why it was working before this change?